Dnd/Dnd.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <!-- Required meta tags -->
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+        <!-- Bootstrap CSS -->
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
+        <link rel="stylesheet" href="../CSS/StyleSheet.css">
+
+        <script src="https://kit.fontawesome.com/f7a97ee3aa.js" crossorigin="anonymous"></script>
+
+        <!-- Website Favicon -->
+        <link rel = "shortcut icon" type = "image/png" href = "../Images/Logo.png">
+
+        <!-- Logo Font -->
+        <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@800&display=swap" rel="stylesheet">
+
+        <title>DevSoc and Dragons</title>
+    </head>
+
+    <body class="dnd-bg" onLoad="displaySeasonContent()">
```

Comment:
Does this work? I thought it had to be lowercase, i.e. `onload="..."`

Reply:
yeah this works